buildkite/scripts/build_webui.sh

```
@@ -2,4 +2,6 @@
 
 set -euo pipefail
 
+export DOCKER_USERNAME=$(buildkite-agent secret get DOCKER_USERNAME)
```

Shellcheck

[warning] Declare and assign separately to avoid masking return values.

(SC2155)